sys/kern/subr_intr.c

```diff
@@ -479,6 +505,12 @@ isrc_free_irq(struct intr_irqsrc *isrc)
 	return (0);
 }
 
+device_t
+intr_irq_root_dev(void)
+{
+	return intr_irq_roots[INTR_ROOT_IRQ].dev;
```

Comment:
Same as above.

Reply:
As above, not since 2019, but return x should always have x parenthesised.

Reply:
oh thanks for pointing this out! I somehow missed the return value convention when going through the style guide

Reply:
fixed